br/pkg/lightning/backend/backend_test.go

```diff
@@ -31,22 +32,25 @@ func Test(t *testing.T) {
 
 // FIXME: Cannot use the real SetUpTest/TearDownTest to set up the mock
 // otherwise the mock error will be ignored.
-
-func (s *backendSuite) setUpTest(c gomock.TestReporter) {
-	s.controller = gomock.NewController(c)
-	s.mockBackend = mock.NewMockBackend(s.controller)
-	s.backend = backend.MakeBackend(s.mockBackend)
-	s.ts = oracle.ComposeTS(time.Now().Unix()*1000, 0)
+func setUpTest(c gomock.TestReporter) *backendSuite {
```

Comment:
```suggestion
func createBackendSuite(c gomock.TestReporter) *backendSuite {
```

Also please remove redundant code snippet above:

```go
var _ = Suite(&backendSuite{})
func Test(t *testing.T) {
	TestingT(t)
}
```

Make sure you understand how tests are running.

Reply:
sorry. i forget to delete it. is my mistake.